package.json

```diff
@@ -1,76 +1,85 @@
 {
-  "name": "ilc-sdk",
-  "version": "5.2.4",
-  "description": "SDK for app development with Isomorphic Layout Composer",
-  "main": "dist/server/index.js",
-  "types": "dist/server/index.d.ts",
-  "files": [
-    "dist/**",
-    "app.js",
-    "app.d.ts"
-  ],
-  "scripts": {
-    "build": "tsc -p ./src/app && tsc -p ./src/server",
-    "test": "mocha -r ts-node/register test/**/*.ts",
-    "coverage": "nyc npm run test",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "format:check": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\"",
-    "lint": "tslint -p ./src/app && tslint -p ./src/server",
-    "docs": "typedoc",
-    "preversion": "npm run format:check && npm run lint && npm test",
-    "prepublish": "npm run build"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/namecheap/ilc-sdk.git"
-  },
-  "author": "Namecheap, Inc",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/namecheap/ilc-sdk/issues"
-  },
-  "homepage": "https://github.com/namecheap/ilc-sdk#readme",
-  "devDependencies": {
-    "@knodes/typedoc-plugin-pages": "^0.23.4",
-    "@types/chai": "^4.3.9",
-    "@types/chai-as-promised": "^7.1.7",
-    "@types/jsdom": "^21.1.4",
-    "@types/lodash.merge": "^4.6.8",
-    "@types/memoizee": "^0.4.10",
-    "@types/mocha": "^10.0.3",
-    "@types/node": "^20.8.7",
-    "@types/sinon": "^10.0.20",
-    "@types/uglify-js": "^3.17.3",
-    "chai": "^4.3.10",
-    "chai-as-promised": "^7.1.1",
-    "jsdom": "^22.1.0",
-    "jsdom-global": "^3.0.2",
-    "mocha": "^10.2.0",
-    "mock-http": "^1.1.0",
-    "nock": "^13.3.6",
-    "nyc": "^15.1.0",
-    "prettier": "^3.0.3",
-    "sinon": "^17.0.0",
-    "ts-node": "^10.9.1",
-    "tslint": "^6.1.2",
-    "tslint-config-prettier": "^1.18.0",
-    "typedoc": "~0.23.28",
-    "typedoc-plugin-missing-exports": "^1.0.0",
-    "typedoc-plugin-rename-defaults": "0.6.6",
-    "typescript": "~5.0.4"
-  },
-  "dependencies": {
-    "@types/url-join": "^4.0.2",
-    "avsc": "^5.7.7",
-    "axios": "^1.5.1",
-    "lodash.merge": "^4.6.2",
-    "memoizee": "^0.4.15",
-    "uglify-js": "^3.17.4",
-    "url-join": "^4.0.1",
-    "webpack-inject-plugin": "^1.5.5",
-    "wrapper-webpack-plugin": "^2.2.2"
-  },
-  "engines": {
-    "npm": ">=7"
-  }
+    "name": "ilc-sdk",
+    "version": "0.0.0",
```

Comment:
I'm not changing version here. It will continue from 5.2.4 (current one).
I'm removing it from package.json to not confuse anyone, because it will not be bumped  in this file (see https://semantic-release.gitbook.io/semantic-release/support/faq#it-is-not-needed-for-semantic-release-to-do-its-job)
If we want to make back-pushes in repositories let me know, it's possible to do but will require to create additional auth app in namecheap org.

Reply:
I see, thanks